Classes/CardIODataEntryViewController.m

```diff
@@ -164,7 +164,14 @@ - (void)viewDidLoad {
     self.cardView.bounds = CGRectZeroWithSize(CGSizeMake((CGFloat)ceil(self.floatingCardView.bounds.size.width),
                                                          (CGFloat)ceil(self.floatingCardView.bounds.size.height)));
     self.cardView.contentMode = UIViewContentModeScaleAspectFit;
-    self.cardView.backgroundColor = kColorViewBackground;
+    
+    if (iOS_13_PLUS) {
```

Comment:
Is this a macro? You could have used the [Availability API](https://developer.apple.com/documentation/swift/objective-c_and_c_code_customization/marking_api_availability_in_objective-c) for this use case.

Reply:
Considered it, but upstream already had this nifty macro setup that they use everywhere. When in Rome...

Reply:
Yeah, I only noticed that later and that was the reason why your PR got approved 😄